apim-traffic-tool/lib/attack-tool/src/python/constants.py

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2019, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+# Constants
+
+GENERAL_CONFIG = 'general_config'
+API_HOST = 'api_host'
+NUMBER_OF_PROCESSES = 'number_of_processes'
+USER_AGENTS = 'user_agents'
+PAYLOADS = 'payloads'
+ATTACK_DURATION = 'attack_duration'
+PORT = 'port'
+IP = 'ip'
+PROTOCOL = 'protocol'
+COMPROMISED_USER_COUNT = 'compromised_user_count'
+STOLEN_TOKEN = 'stolen_token'
+ATTACKS = 'attacks'
+TIME_PATTERNS = 'time_patterns'
+
+
+
+
```

Comment:
Remove these empty lines

Reply:
Removed empty lines except for the last line since PEP 8 recommend the one blank line at the end of the source file.